src/randomizer.py

```diff
@@ -83,6 +86,37 @@
 
 T = TypeVar("T")
 
+method_list: List[str] = [
```

Comment:
METHOD_LIST since it's global. Let's make a list of functions instead of strings, to get improved type safety and eliminate the need for using globals(). Decorators could be neat.

Reply:
done